db/data.py

```diff
@@ -414,6 +426,55 @@ def load_high_level(mbid, offset=0):
         return {"metadata": metadata, "highlevel": highlevel}
 
 
+def model_class_mappings():
+    """Fetches key mappings of model classes from database
+
+       Returns:
+           data: dict containing descriptive keys for model classes
+    """
+    mappings = {}
+
+    with db.engine.connect() as connection:
+        query = text("""SELECT model.model, mappings
+                          FROM model
+                         WHERE mappings is not null
+        """)
+        result = connection.execute(query)
+        key_map_list = result.fetchall()
+
+    for row in key_map_list:
+        if key_map_list is None:
+            continue
+        mappings[row["model"]] = row["mappings"]
+
+    return mappings
+
+
+def map_class_labels(high_level_dict):
+    """Maps shortend key names in high-level dict to meaningful labels using
+    mappings returned by `model_class_mappings` 
+
+    Args:
+        high_level_data: high-level data dict containing shortend keys
+    
+    Returns:
+        dict with mapped model class keys
+
+    """
+    mappings = db.data.model_class_mappings()
+    high_level = high_level_dict["highlevel"]
+
+    for model, map in mappings.iteritems():
+        for key, val in map.iteritems():
```

Comment:
This is a small thing, but I would do split this into 2 methods and do these loops the other way around. Instead of looping through the mapping and changing the highlevel data, loop through the highlevel data and look at the mapping data.
Also, it may result in a bit more duplicate code, but if you pass the mapping into the function as well then you can perform tests without having to access the database:

```py

def map_highlevel_class_labels(highlevel, mappings):
   for model, data in highlevel.items():
        if model in mappings:
          map_single_highlevel_class_labels(data, mappings[model])
         #construct return value

def map_single_highelvel_class_labels(data, mapping):
    value = data["value"]
    data["value"] = mapping.get(value, value)
    all = {}
    for k, v in data["all"]:
        all[mapping.get(k, k)] = v
    data["all"] = all
    return data
```
(not tested)
Make sure the methods also have correct docstrings.
It's a bit longer, but I think it allows us to consider the workflow in terms of processing the data, rather than processing the mapping. Now you can also test them without having to add the data to the database first, instead you can create a generic mapping directory and test that the methods work.